cmd/lsif-go/args.go

```diff
@@ -50,7 +50,7 @@ func init() {
 	// Verbosity options
 	app.Flag("quiet", "Do not output to stdout or stderr.").Short('q').Default("false").BoolVar(&noOutput)
 	app.Flag("verbose", "Output debug logs.").Short('v').CounterVar(&verbosity)
-	app.Flag("no-animation", "Do not animate output.").Default("false").BoolVar(&noAnimation)
+	app.Flag("no-animation", "Do not animate output.").Default("true").BoolVar(&noAnimation)
```

Comment:
I wasn't able to figure out how to set `no-animation` to true if you want to force animation. I don't consider it a blocking issue for this PR, but it would be nice to allow users to opt-into animation (although I doubt anyone will bother).

Reply:
You won't believe this (I had to do a double take) but apparently the library used here, `kingping`, [creates a `--no-<flag>` complement for each boolean flag](https://github.com/alecthomas/kingpin#boolean-values), so... yes:

```
./lsif-go --no-no-animation
```

Reply:
Very nice. This is quite common among cli parser libraries, but I didn't give it a try. Just updated the flag to allow `--animation`